webapp/pages/profile/_id/_slug.vue

```diff
@@ -92,140 +56,72 @@
               v-if="!myProfile"
               :follow-id="user.id"
               :is-followed="user.followedByCurrentUser"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-15T14:58:37Z" title="Wednesday, May 15th 2019, 4:58:37 pm +02:00">May 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -308,18 +318,32 @@
               />
             </ds-flex-item>
           </template>
+
+
         </ds-flex>
         <hc-load-more
           v-if="hasMore"
           :loading="$apollo.loading"
           @click="showMoreContributions"
         />
       </ds-flex-item>
+
+
+
+      
```


@ogerly don't add extra lines if not really necessary. At least don't add 2 or more new lines.